packages/core/src/TxBuilders/TxBuilder.V3.class.ts

```diff
@@ -502,23 +491,19 @@ export class TxBuilderV3 extends TxBuilderAbstractV3 {
       tx.lockAssets(
         address,
         makeValue(ORDER_DEPOSIT_DEFAULT, [poolRefAssetIdHex, 1n]),
-        Data.void(),
+        Void(),
```

Comment:
In v1 builder (I think) you used a constant `VOID` for `Data.void()`, but here you use the exported `Void()`. Is that intended?

Reply:
VOID is used where previously we used Data.void().toCbor(). Here Void() matches Data.void()